Core/include/Acts/Propagator/ActorList.hpp

```diff
@@ -0,0 +1,130 @@
+  ///
+  /// @param actors The source action list
+  ActorList(const ActorList<actors_t...>& actors) = default;
+
+  /// Default move constructor
+  ///
+  /// @param actors The source action list
+  ActorList(ActorList<actors_t...>&& actors) = default;
+
+  /// Default move assignment operator
+  ///
+  /// @param actors The source action list
+  ActorList<actors_t...>& operator=(const ActorList<actors_t...>& actors) =
+      default;
+
+  /// Default move assignment operator
+  ///
+  /// @param actors The source action list
+  ActorList<actors_t...>& operator=(ActorList<actors_t...>&& actors) = default;
+
+  /// Constructor from tuple
+  ///
+  /// @param actors Source extensions tuple
+  ActorList(const std::tuple<actors_t...>& actors)
+      : detail::Extendable<actors_t...>(actors) {}
+
+  /// Constructor from tuple move
+  ///
+  /// @param actors Source extensions tuple
+  ActorList(std::tuple<actors_t...>&& actors)
+      : detail::Extendable<actors_t...>(std::move(actors)) {}
+
+  /// Append new entries and return a new condition
+  template <typename... appendices_t>
+  ActorList<actors_t..., appendices_t...> append(appendices_t... aps) const {
+    auto catTuple =
+        std::tuple_cat(tuple(), std::tuple<appendices_t...>(aps...));
+    return ActorList<actors_t..., appendices_t...>(std::move(catTuple));
+  }
+
+  /// Act call which broadcasts the call to the tuple() members of the list
+  ///
+  /// @tparam propagator_state_t is the state type of the propagator
+  /// @tparam stepper_t Type of the stepper used for the propagation
+  /// @tparam navigator_t Type of the navigator used for the propagation
+  ///
+  /// @param [in,out] state This is the propagator state object
+  /// @param [in] stepper The stepper in use
+  /// @param [in] navigator The navigator in use
+  /// @param [in] args The arguments to be passed to the actions
+  template <typename propagator_state_t, typename stepper_t,
+            typename navigator_t, typename... Args>
+  void act(propagator_state_t& state, const stepper_t& stepper,
+           const navigator_t& navigator, Args&&... args) const {
+    using impl = detail::actor_list_impl<actors_t...>;
+    impl::act(tuple(), state, stepper, navigator, std::forward<Args>(args)...);
+  }
+
+  /// Check call which broadcasts the call to the tuple() members of the list
+  ///
+  /// @tparam propagator_state_t is the state type of the propagator
+  /// @tparam stepper_t Type of the stepper
+  /// @tparam navigator_t Type of the navigator
+  ///
+  /// @param [in,out] state is the state object from the propagator
+  /// @param [in] stepper Stepper used for the propagation
+  /// @param [in] navigator Navigator used for the propagation
+  /// @param [in] args are the arguments to be passed to the aborters
+  template <typename propagator_state_t, typename stepper_t,
+            typename navigator_t, typename... Args>
+  bool check(propagator_state_t& state, const stepper_t& stepper,
```

Comment:
I think I would lean slightly towards naming this function actually `abort`, because `check` is very generic...

Reply:
I am also not quite happy with `act` and even less with `check`. `abort` sounded to me like we would call for an abort. Maybe `checkAbort`?

Reply:
`checkAbort` sounds good